lib/extensions/isolate-sim.js

```diff
@@ -0,0 +1,23 @@
+import { getDevices, deleteDevice } from 'node-simctl';
+import log from '../logger';
+import _ from 'lodash';
+
+
+let extensions = {};
+
+extensions.isolateSim = async function () {
```

Comment:
Why do we have this function anyways? can we document why it exists?
Something tells me it's not actually very effective, or is not quite them best way to solve a problem.